components/atoms/LanguagePill/LanguagePill.tsx

```diff
@@ -64,7 +64,7 @@ const LanguagePill = ({ topic, classNames, onClick }: LanguagePillProps) => {
   return (
     <div
       onClick={onClick}
-      className={`cursor-pointer text-xs text-light-slate-11 rounded-[1.875rem] w-max flex items-center gap-1 py-2 px-4 bg-light-slate-6 ${
+      className={`cursor-pointer text-xs  rounded-[1.875rem] w-max flex items-center gap-1 py-2 px-4 bg-light-slate-6 ${
```

Comment:
IMHO we need to keep this class because this component we used in multiple places.

Reply:
Thank you for your valuable feedback! I understand your point about the Component being used in multiple places, 
 so i have used an alternative approach to fix the issue and I appreciate your warm welcome to the community.